layouts/index.html

```diff
@@ -193,16 +193,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
                     <br>
                     <p><strong>2.</strong> Then quick install RAPIDS with:</p>
                     <pre
-                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-23.12 -c rapidsai -c conda-forge -c nvidia rapids=23.12 python=3.10 cudatoolkit=11.8</code></pre>
+                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-24.02 -c rapidsai -c conda-forge -c nvidia rapids=24.02 python=3.10 cudatoolkit=11.8</code></pre>
```

Comment:
Can we switch to CUDA 12 here? We already use CUDA 12 in our pip wheels.
```suggestion
                        class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-24.02 -c rapidsai -c conda-forge -c nvidia rapids=24.02 python=3.10 cuda-version=12.0</code></pre>
```